File: 1-easy/problems/1848-minimum-distance-to-the-target-element/solution.ts
function getMinDistance(nums: any, target: any, start: any): boolean | number | string | any {
  """
  :type nums: List[int]
  :type target: int
  :type start: int
  :rtype: int
  """
  for (i in Array.from({length: (nums}, (_, i) => i).length)) {
      if (start-i >= 0 && nums[start-i] == target) || \
         (start+i < (nums).length && nums[start+i] == target):
          break
  return i;
}

export { getMinDistance };

File: 1-easy/problems/1848-minimum-distance-to-the-target-element/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def getMinDistance(self, nums, target, start):
        """
        :type nums: List[int]
        :type target: int
        :type start: int
        :rtype: int
        """
        for i in range(len(nums)):
            if (start-i >= 0 and nums[start-i] == target) or \
               (start+i < len(nums) and nums[start+i] == target):
                break
        return i
